src/components/AddressSearch.js

```diff
@@ -130,7 +132,10 @@ const AddressSearch = (props) => {
 
         const values = {
             street: props.value ? props.value.trim() : '',
-            city: city || cityFallback,
+            // When locality is not returned, many (non-US) countries return the city as
+            // postalTown, otherwise as sublocality. If postalTown is returned, the
+            // sublocality will be a city subdivision so wouldn't be appropriate.
```

Comment:
Can we add in the comment examples of addresses for each case? 🙏

Reply:
Ah yes, good idea! Added now.